File: app/src/main/java/com/ragazm/noteapplicationtest/MyViewHolder.java
package com.ragazm.noteapplicationtest;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

//import com.ragazm.noteapplicationtest.databinding.RowLayoutBinding;

/**
 * Created by Andris on 022 22.10.17.
 * TODO implement data binding
 */

public class MyViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {

   // RowLayoutBinding rowLayoutBinding;


    TextView txtTitle;
    TextView txtText;
   

    public MyViewHolder(View itemView) {
        super(itemView);
        txtTitle = (TextView)itemView.findViewById(R.id.txtText);
        txtText = (TextView)itemView.findViewById(R.id.txtTitle);
        //itemView.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {

    }
}

File: app/src/main/java/com/ragazm/notepad/recycler/MyViewHolder.kt
package com.ragazm.notepad.recycler

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import com.ragazm.notepad.R



/**
 * Created by Andris on 022 22.10.17.
 *
 */

class MyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView), View.OnClickListener {

    // RowLayoutBinding rowLayoutBinding;


    internal var txtTitle: TextView = itemView.findViewById(R.id.txtText) as TextView
    internal var txtText: TextView = itemView.findViewById(R.id.txtTitle) as TextView




    override fun onClick(v: View) {

    }
}
